src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -395,10 +398,25 @@ class LicenceService(
       ?: throw ValidationException("Staff with username $username not found")
 
     val updatedLicence = when (licenceEntity) {
-      is CrdLicence -> licenceEntity.submit(submitter as CommunityOffenderManager)
+      is CrdLicence -> {
+        if (!licenceHolderIsEligible(licenceEntity.nomsId)) {
+          throw ValidationException("Can only submit cases eligible for CVL")
```

Comment:
400 response is fine.. could do a 409 conflict but it's probably not worth the faff considering this is the internal API and we don't need to distinguish between the two types of failure.